pages/posts/first-post.js

```diff
@@ -13,12 +20,21 @@ export default function FirstPost() {
         </Head>
         <div className='flex justify-center pt-9 pb-5 bg-zinc-50'>
           <div className='grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-6'>
-            <HouseCard />
-            <HouseCard />
-            <HouseCard />
-            <HouseCard />
-            <HouseCard />
-            <HouseCard />
+            {data.map((card) => {
+              return (
+                <HouseCard
```

Comment:
Está faltando a key https://pt-br.reactjs.org/docs/lists-and-keys.html#keys

Reply:
Feito!